crates/app/tests/helpers/tree.rs

```diff
@@ -79,3 +82,21 @@ pub async fn create_path(
         res.text().await.unwrap()
     );
 }
+
+pub async fn get_path(
+    cl: &reqwest::Client,
+    addr: &str,
+    repo: &RepositoryName,
+    tag: &TagName,
+    path: &TreePath,
+) -> Vec<u8> {
+    let url = format!("{}/{}/_tag/{}/tree{}", addr, repo, tag, path);
+
+    let res = cl.get(&url).send().await.unwrap();
+    assert_eq!(res.status(), StatusCode::OK);
```

Comment:
As discussed offline, check `Content-Length` and `Content-Digest` header values and return the `Content-Type` value